test/distributed/org/apache/cassandra/distributed/impl/AbstractCluster.java

```diff
@@ -477,6 +477,11 @@ public ICoordinator coordinator(int node)
         return instances.get(node - 1).coordinator();
     }
 
+    public List<ICoordinator> coordinators()
```

Comment:
Should we return `stream` as we do with, say, instances here below?

Reply:
We can return a `Stream`, yes. Not sure which one is going to be more useful, though.